threejs/fxhash_Rooms/src/figures.js

```diff
@@ -209,9 +208,65 @@ const genererFigures = (fxhash) => {
       hatch: true, // Fill with white texture
       full: false, // Fill with color texture (in the anaverse, red and cyan)
     });
+
+    // "create" the vertical column members to extend through the ground plane
+    figures.push({
+      geometry: { type: "BoxGeometry", args: [0.1, 20, 0.1] },
+      pos: {
+        // Anaglypic Question
+        // Can we add a column that runs from each position of the original room polyline down through the ground plane? 
+        // Position for each beam should be the position of each splinePt used to create the polygon from the floor generator
+        x: px * u, 
+        y: py * u - 20/2, // <---- Arbitrary column depth should match the column height and should pass through the ground plane
+        z: pz * u,
```

Comment:
Hey @Anaglyph-ic I could use a bit of help figuring out the placement of each of these columns. They should be placed on each position[i] x,y,z 

As noted, the columns should extend through the ground plane (see next comment). So this can be an arbitrary depth so long as they pass through the ground and are not visible.

Reply:
There is no ground in the anaverse, if you fall, you fall towards nil and go back to an origin position (0,1,0).

Reply:
To place the columns, you need to iterate over the roomPos array which contains the polyline. This code is a bit more complicated than just the simplest array because the shape drawing can have operations that more complex than the simple line (like bezierTo, etc). Hence the need for the `drawArgs` property.

Your code then becomes something like:

```js

roomPos.forEach(rP=>{

 figures.push({
      geometry: { type: "BoxGeometry", args: [0.1, 20, 0.1] },
      pos: {
        // Anaglyphic Answer
        // Position using the roomPos array, that is the polygon from the floor generator
        x: (rP.drawArgs[0] + px) * u,   // here
        y: py * u - 20/2, // <---- Arbitrary column depth should match the column height and should pass through the ground plane
        z: (rP.drawArgs[1]+ pz) * u,   // there
…
```

Reply:
Works perfectly! Thanks!